scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -13,19 +13,36 @@
    "cell_type": "markdown",
```

Comment:
<p>Perhaps some of this material can be moved to the FAQ, and be made more succinct here:</p><p> </p><p><s>We can now</s> <strong>Here we</strong> visualize the training losses computed in each training epoch<s>, and see how the model improves with training. We want to run the training for an appropriate </s><code><s>num_epochs</s></code><s> and </s><code><s>lr</s></code><s> (to be fine-tuned by the user) that produces a loss-curve that tails off. The loss-curve for our training is as follows:</s></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>